### PR TITLE
Updates/ban wallet

### PR DESCRIPTION
**Pull request by lovetrading10**

### CODE REVIEW

File: contracts/core/SettingsManager.sol [hunk 1]
```diff
@@ -122,6 +122,19 @@ contract SettingsManager is ISettingsManager, Ownable, Constants {
         }
     }
 
+    function addDelegatesToBlackList(address[] memory _delegates) external {
+        require(operators.getOperatorLevel(msg.sender) >= uint8(1), "Invalid operator");
+        for (uint256 i = 0; i < _delegates.length; ++i) {
+            EnumerableSet.add(banWalletList, _delegates[i]);
+        }
+    }
+    function removeDelegatesToBlackList(address[] memory _delegates) external {
```
Comment (mookim-eth) [inline contracts/core/SettingsManager.sol]: consider rename removeDelegatesToBlackList to removeDelegatesFromBlackList

File: contracts/core/SettingsManager.sol [hunk 2]
```diff
@@ -468,6 +481,10 @@ contract SettingsManager is ISettingsManager, Ownable, Constants {
             ) / positionVault.poolAmounts(_token, _isLong);
     }
 
+    function checkBlackList(address _delegate) public view override returns (bool) {
```
Comment (mookim-eth) [inline contracts/core/SettingsManager.sol]: do we need to take care of political correctness and change to checkBanList?

**Review by mookim-eth — COMMENTED** [review]
after the ban, can the banner bypass it by creating a delegation to another wallet, then stake/withdraw?

File: contracts/core/SettingsManager.sol (hunk 1 above)
Comment (lovetrading10) [reply contracts/core/SettingsManager.sol]: right

File: contracts/core/SettingsManager.sol (hunk 2 above)
Comment (lovetrading10) [reply contracts/core/SettingsManager.sol]: sure

**Review by mookim-eth — APPROVED** [review]

**Pull request merged**